### PR TITLE
REST functionalities

### PR DESCRIPTION
**Pull request by mmokgethi**

### CODE REVIEW

File: src/main/java/com/bbd/modisa/data/entities/User.java [hunk 1]
```diff
@@ -1,11 +1,13 @@
 package com.bbd.modisa.data.entities;
 
 import javax.persistence.*;
+import javax.xml.bind.annotation.XmlRootElement;
 import java.io.Serializable;
 import java.util.Set;
 import java.util.TreeSet;
 
 @Entity
```
Comment (claido) [inline src/main/java/com/bbd/modisa/data/entities/User.java]: what is this used for?

File: src/main/java/com/bbd/modisa/rest/BankRest.java [hunk 2]
```diff
@@ -63,7 +63,7 @@ public void getBalance(Integer userId){
     @GET
     @Path("/getStatement/{userId}")
     @Produces(MediaType.APPLICATION_JSON)
-    public List<Transaction> getStatement(@PathParam("userId") Integer userId){
+    public Transaction getStatement(@PathParam("userId") Integer userId){
```
Comment (claido) [inline src/main/java/com/bbd/modisa/rest/BankRest.java]: Exposing an entity in the rest layer

File: src/main/java/com/bbd/modisa/data/entities/User.java (hunk 1 above)
Comment (mmokgethi) [reply src/main/java/com/bbd/modisa/data/entities/User.java]: I was representing My values with XML, I used it before you told Me to use JSON...I forgot to remove it...it has no use

File: src/main/java/com/bbd/modisa/service/AccountServiceImpl.java [hunk 3]
```diff
@@ -61,13 +64,17 @@ public double withdraw(double withdrawalAmount, Integer accountId) {
     }
 
     @Override
-    public Transaction getTransactions(Integer userId){
+    public TransactionDto getTransactions(Integer userId){
+        Transaction transaction = new Transaction();
```
Comment (claido) [inline src/main/java/com/bbd/modisa/service/AccountServiceImpl.java]: you dont need this

**Review by claido — APPROVED** [review]

**Pull request merged**